### PR TITLE
Apps images generator

### PR DESCRIPTION
**Pull request by sadov**
Toolset for application images generation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,4 +53,5 @@ The current sub-commands are:
    * `infowatch` -- Info-files monitoring tool with regular expression quering to json fields;
    * `log` -- scans Log file accordingly by regular expression of requests to json fields;
    * `logwatch` -- Log-files monitoring tool with regular expression quering to json fields;
-   * `server` -- micro HTTP-server for providing of baseOS and Apps images.
+   * `server` -- micro HTTP-server for providing of baseOS and Apps images;
+   * `ociimage` -- generate oci image for EVE from local or remote docker image.
```
Comment (deitch) [inline README.md:57]: What does this mean? And oci image _is_ (well, almost _is_, some small differences in spec, but let's not quibble over those) a docker image. It looks like it generates a tar file from the image. Why do we need it?

Comment (giggsoff) [reply README.md:57]: This utility allows you to convert an image from a local or remote docker registry into tar, which can be loaded into EVE from the HTTP data store type (from eden server).

Comment (deitch) [reply README.md:57]: Got it. So it is the equivalent of `docker pull someimage` followed by `docker save someimage > file.tar`. Let's just change the comment then as "save oci image from local or remote registry to tar file for consumption by eve" or something like that.

Comment (giggsoff) [reply README.md]: Made an edit, thanks. 
As an additional clarification: ociimage performs additional operations with the tar archive, similar to [zedUpload](https://github.com/lf-edge/eve/blob/master/pkg/pillar/zedUpload/datastore_oci.go).

**Pull request merged**